pkg/testcase/upgradenodereplacement.go

```diff
@@ -46,7 +46,7 @@ func TestUpgradeReplaceNode(cluster *shared.Cluster, version string) {
 	shared.LogLevel("info", "Scp files to new server nodes done\n")
 
 	serverLeaderIP := cluster.ServerIPs[0]
```

Comment:
qq… we would also need to assign the new external ips to the cluster.ServerIPs, isnt't it? else it would show the previous ips...